Cargo.toml

```diff
@@ -64,12 +64,13 @@ spin-oci = { path = "crates/oci" }
 spin-plugins = { path = "crates/plugins" }
 spin-runtime-factors = { path = "crates/runtime-factors" }
 spin-telemetry = { path = "crates/telemetry", features = [
-  "tracing-log-compat",
+    "tracing-log-compat",
```

Comment:
There's a few of these unrelated layout changes - maybe turn off auto formatting for this file?